src/domain/tokens/__tests__/token.factory.ts

```diff
@@ -0,0 +1,20 @@
+import { faker } from '@faker-js/faker';
+import { Token, TokenType } from '../entities/token.entity';
+
+export default function (
+  address?: string,
+  decimals?: number,
```

Comment:
If `decimals` is null we should allow that here too.

Reply:
Changed in [6d58c66](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/6d58c665e46105677aa5f746b9ac461b4e0c8ee3)